pkg/api/validation/validation.go

```diff
@@ -397,6 +401,20 @@ func validateISCSIVolumeSource(iscsi *api.ISCSIVolumeSource) errs.ValidationErro
 	return allErrs
 }
 
+func validateFCVolumeSource(fc *api.FCVolumeSource) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	if len(fc.TargetWWNs) < 1 {
+		allErrs = append(allErrs, errs.NewFieldRequired("targetWWNs"))
+	}
+	if fc.FSType == "" {
+		allErrs = append(allErrs, errs.NewFieldRequired("fsType"))
+	}
+	if fc.Lun < 0 || fc.Lun > 255 {
```

Comment:
[`lun`](https://github.com/kubernetes/kubernetes/pull/12599/files#diff-e58c0c0709cf6f8a3b00aef6069b66b7R703) doesn't allow `omitempty`, so user has to be provide a valid `lun`.

Just to verified a yaml with empty `lun`, the follow error came up:
```text
error validating "./fc.yaml": error validating data: field lun: is required; if you choose to ignore these errors, turn validation off with --validate=false
```